src/simulation.jl

```diff
@@ -175,12 +176,12 @@ function HorizontalConvectionSimulation(; Ra=1e11, h₀_frac=0.6, output_writer=
     
     χ = @at (Center, Center, Center) κ * (∂x(b)^2 + ∂z(b)^2)
     
-    ke = @at (Center, Center, Center) 1/2 * (u^2 + v^2 + w^2)
+    ke = @at (Center, Center, Center) 1/2 * (u^2 + w^2)
```

Comment:
There can still be a meridional velocity in a flat simulation! It just doesn't vary in `y`.